### PR TITLE
feat(create-redwood-app): cli flags to show up help and version information

### PR DESCRIPTION
**Pull request by jamesgeorge007**
```sh
create-redwood-app --help | -h
create-redwood-app --version | -v
```

Previously `create-redwood-app --help` would create a directory under the name `--help` within the current path.

### CODE REVIEW

**Comment by thedavidprice** [comment]
@jamesgeorge007 thanks for this! And great catch. This will be 100% needed sooner than later, for example, once the TypeScript support is complete and individuals can choose the installation language target.

Your custom code looks great. But we've been using [Yargs](https://github.com/yargs/yargs) for the Redwood CLI ([example file](https://github.com/redwoodjs/redwood/blob/master/packages/cli/src/index.js)). Would you be interested in attempting to implement this with Yargs? 

Definitely open to discussion! And, at the very least, we could go ahead and move forward with merging this code and then following up with a Yargs improvement.

Thoughts?

**Comment by jamesgeorge007** [comment]
Maybe we could onboard an arg-parser library, but IMHO it isn't necessary for a CLI utility of similar sort whose sole purpose is project scaffolding.

**Comment by thedavidprice** [comment]
At this time, the requirements are very simple indeed. But in the near future there are going to be increasing options for the type of installation:
- language target (as I mentioned)
- Sides to include: default will be web and api but installation might target only one or the other, and in the future, we'll see react native, electron, etc. be available
- Templates: this is a bit further out, but using a different "starter" or configuration will be technically possible (although unplanned as of yet)
- Tying in other tools (like diagnostics) at the time of installation, etc.

The last reason is for consistency -- we're using Yargs everywhere else. Whether or not we do it now, we _will_ end up using Yargs for this.

All that said, I'm ok to proceed as is. But if you'd like to take a swing at Yargs, it's definitely of value and I (we) would support you as needed.

What would you like to do as a next step?

**Comment by jamesgeorge007** [comment]
Let's land this one as of now and revisit the `yargs` implementation in a future PR.

**Comment by thedavidprice** [comment]
@jamesgeorge007 sounds good and the PR looks good. Merging now. 

Thanks again 🚀

File: packages/create-redwood-app/src/create-redwood-app.js [hunk 1]
```diff
@@ -38,7 +40,22 @@ const downloadFile = async (sourceUrl, targetFile) => {
   })
 }
 
-const targetDir = String(process.argv.slice(2)).replace(/,/g, '-')
+const [, , ...args] = process.argv
```
Comment (peterp) [inline packages/create-redwood-app/src/create-redwood-app.js:43]: @thedavidprice We should clean this up ASAP

**Pull request merged**